node/src/components/consensus/era_supervisor.rs

```diff
@@ -696,6 +697,28 @@ where
                 .collect(),
         }
     }
+
+    /// Emits a fatal error if the consensus state is still empty.
+    pub(super) fn shutdown_if_necessary(&self) -> Effects<Event<I>> {
```

Comment:
Does this ever return an effect?
```suggestion
    pub(super) fn shutdown_if_necessary(&self) {
```

Reply:
`fatal` is an effect. See Mateusz's comment above.

Reply:
Ah, sorry! :+1: